dc_motor:
 channel: 1
 pwm_min: 220
 pwm_max: 410
 pwm_midpoint: 315
 pwm_midrange_min: 280
 pwm_midrange_max: 330
servo:
 channel: 0
 pwm_min: 205
 pwm_max: 410
 pwm_midpoint: 308
